dspace/config/crosswalks/orcid/mapConverter-orcid-to-dspace-publication-type.properties

```diff
@@ -0,0 +1,11 @@
+website = Other
```

Comment:
Please add a comment at the top of this file to describe the key/value.  I'm assuming this is essentially the opposite of `mapConverter-dspace-to-orcid-publication-type.properties`?